content/stands/gnome.md

```diff
@@ -0,0 +1,29 @@
+---
+description: Learn about the latest developments in the GNOME desktop environment! At the GNOME booth, you can check out live demos of GNOME 3.38 and GTK 4, get help using GNOME, and learn more about paid GNOME internship opportunities. During our themed chat hours, you'll be able to engage and talk directly with GNOME developers, designers, Board of Directors members, Foundation staff, and other community members. Ask us your questions or just pop in to say "Hi" and chat! Take a look at our chat schedule - available at our booth - to see what our topics are and when you can join them.
+layout: stand
+logo: stands/gnome/logo.png
```

Comment:
Could you move the logo directly to the stands folder? No need to create a new folder called "gnome" the logo could just be called "gnome.png"

Reply:
Okayy, will name it as gnome-logo.png